.github/workflows/ci.yml

```diff
@@ -99,7 +99,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [13, 12, 10, 8, 6]
+        node-version: [14, 13, 12, 10, 8, 6]
```

Comment:
It seems to latest node on GitHub actions is still 14: https://github.com/babel/babel/runs/1298133296#step:3:7